igcollect/switch.py

```diff
@@ -223,6 +223,8 @@ def get_switch_model(snmp):
         return 'powerconnect'
     elif 'ProCurve' in model:
         return 'procurve'
+    elif 'Aruba' in model:
+        return 'procurve'
```

Comment:
Might be a stupid question, but should that really return `procurve` in case of `Aruba`?

Reply:
Nah, they are basically identical switches.